py/server/deephaven/stream/kafka/consumer.py

```diff
@@ -280,6 +283,55 @@ def _consume(
     except Exception as e:
         raise DHError(e, "failed to consume a Kafka stream.") from e
 
+def protobuf_spec(
+        schema: str,
+        schema_version: Optional[int] = None,
+        schema_message_name: Optional[str] = None,
+        include: Optional[List[str]] = None,
+) -> KeyValueSpec:
+    """The kafka protobuf specs. This will fetch the protobuf descriptor for the schema subject from the schema registry
+    using version schema_version and create protobuf message parsing functions according to parsing options. These
+    functions will be adapted to handle schema changes.
```

Comment:
```suggestion
    """Creates a spec for how to use Kafka data when consuming a Kafka stream to a Deephaven table. This will fetch the protobuf descriptor for the schema subject from the schema registry
    using version schema_version and create protobuf message parsing functions according to parsing options. These
    functions will be adapted to handle schema changes.
```
Changed to be consistent with other docs.